tests/datasets/api_tests.py

```diff
@@ -864,13 +865,15 @@ def test_export_dataset(self):
             back_references=False,
             include_defaults=False,
         )
+        print(cli_export)
```

Comment:
`dd`

Reply:
I added that to debug, since I'm having trouble reproduce the error locally. Will remove after.